packaging/linux/openra.6

```diff
@@ -0,0 +1,68 @@
+.TH OPENRA 6
+.SH NAME
+openra \- An open-source implementation of the Command & Conquer: Red Alert engine using .NET/Mono and OpenGL.
```

Comment:
OpenRA isn't an implementation of the C&C:RA engine.  This is just asking for trouble from those who are angry that OpenRA isn't a drop-in bug compatible reimplementation.